flow360/component/simulation/models/volume_models.py

```diff
@@ -315,6 +315,13 @@ class Fluid(PDEModelBase):
         )
     )
 
+    relative_interpolation_tolerance: Optional[pd.PositiveFloat] = pd.Field(
+        None,
+        description="Interopolation will fail if the distance between an interpolation "
+        "point and the closest triangle is greater than `relative_interpolation_tolerance` "
+        "multiplied by the maximum edge length of the patch containing the interpolation point. "
+        "If not specified, default value is 0.2.",
+    )
```

Comment:
1. instead of defaulting to 0.2 in the solver, should we also default it to 0.2 here?
2. maybe we can call it `interface_relative_interpolation_tolerance` or similar to indicate it is not general interpolation tolerance but at interfaces.
3. Also, should also `Rotation` and `Solid` models have `relative_interpolation_tolerance` parameter?